src/main/java/org/apache/commons/imaging/palette/ColorSpaceSubset.java

```diff
@@ -24,8 +24,8 @@
 
     private static final Logger LOGGER = Logger.getLogger(ColorSpaceSubset.class.getName());
 
-    final int[] mins;
-    final int[] maxs;
+    final int[] min;
+    final int[] max;
```

Comment:
I think these were supposed to be arrays with minimums and maximums. It's common to use the "maximums" or "minimums" as the "locals minimums" for example. So `mins` is a better variable name IMO, so that without seeing the type you know it's an array (especially given it's in a class related to color space).

Reply:
HI @kinow 
Sounds logical. Changed
TY